### PR TITLE
circleci: also test example.c from examples

### PR DESCRIPTION
**Pull request by axic**

### CODE REVIEW

**Comment by axic** [comment]
@chfast any idea why example is failing?

File: examples/example.c [hunk 1]
```diff
@@ -34,9 +34,11 @@ int main()
     msg.depth = 0;
     struct evmc_result result = evmc_execute(vm, ctx, EVMC_HOMESTEAD, &msg, code, code_size);
     printf("Execution result:\n");
+    int exit_code = 0;
     if (result.status_code != EVMC_SUCCESS)
     {
         printf("  EVM execution failure: %d\n", result.status_code);
+        exit_code = -1;
```
Comment (chfast) [inline examples/example.c]: It returns this.

Comment (axic) [reply examples/example.c]: But it shouldn't :)

Comment (axic) [reply examples/example.c]: Based on the inputs in this file, this case should be triggered: https://github.com/ethereum/evmc/blob/master/examples/example_vm/example_vm.c#L119

Comment (chfast) [reply examples/example.c]: `code_size` is wrong.

**Pull request merged**